CMakeLists.txt

```diff
@@ -14,6 +14,9 @@ project(
 	DESCRIPTION "Extensible Combinatorial Optimization Learning Environments"
 )
 
+# path to e.g. findNumPy module
+set(CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake/Modules)
```

Comment:
I'm not sure we want to add this. It overrides CMake search path, which may lead to problems outside of your use case.